lib/v2/screens/dashboard/dashboard.dart

```diff
@@ -43,6 +42,8 @@ class Dashboard extends StatefulWidget {
 class _DashboardState extends State<Dashboard> {
```

Comment:
this dashboard class also stateful?

Reply:
oh i see, you moved these from v1 to v2. 

Please dont move classes over to v2 unless they are fully conformance to v2 standards. 
This class probably should not be stateful. and many other things that come with it